### PR TITLE
Merging Project 3 code into main

### PR DESCRIPTION
**Pull request by simpliweb**
MVP
- getting API call result a list of cocktails
- grab specific data from API call and display on page
- user can get a random cocktail suggestion by clicking a ‘surprise’ button
- user can click ‘heart’ icon on cocktail image container to save the result as ‘favorites’ in firebase
- saved ‘favorites’ display underneath random cocktail result container

STRETCH GOALS (incomplete Q_Q)
- saved 'favorite cocktails' to flow up stream endlessly in the background
- add ingredients

INCOMPLETE
- heart button need to go back to original (unclicked/empty heart) once clicked
- no error handling in API call
- not responsive (e.g., heart location not fixed in one position, rest of the page not responsive)
- pseudo code N/A
- skip link not to the top Q_Q

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,157 @@
...
+
+* { -moz-box-sizing: border-box; -webkit-box-sizing: border-box; box-sizing: border-box; }
+
+.clearfix:after {visibility: hidden; display: block; font-size: 0; content: ''; clear: both; height: 0; }
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
+.skipLink {
```
Comment (michedmnt) [inline src/App.css:9]: SkipLink right now covers the title, would need to adjust the height of the SkipLink and possibly the color for those that are color blind, and unable to see the bright Red Color.

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,157 @@
...
+    margin: 0 auto;
+    border-radius: 5px;
+    background-color: rgb(121, 248, 231);
+    color: rgb(253, 6, 6);
+    font-weight: bold;
 }
 
-.App-link {
-  color: #61dafb;
+footer {
```
Comment (michedmnt) [inline src/App.css:61]: As we discussed. Footer position is kind of in the middle of nowhere, perhaps top right of App would be better suited given you said you will be having an endless list of drinks.

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const Header = () => {
+    return (
+        <header>
+            <h1 className="title">Your Favorite Cocktail 🍷</h1>
```
Comment (michedmnt) [inline src/components/Header.js:4]: possibly create another div within the header to have instructions for the site. 📃

File: src/components/FavoriteCocktail.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                        ? <i class="fa-solid fa-heart"></i>
+                        : <i class="fa-regular fa-heart"></i>
+                    }
+                </button>
+            </div>
+
+            <div className="savedCocktail">
+                {
+                    cocktails.map( (cocktail) => {
+                            return <div className="savedCocktailContent">
```
Comment (pocket0079) [inline src/components/FavoriteCocktail.js]: Great app! Maybe have a way to delete cocktails from favourites

**Review by jaewanbae — COMMENTED** [review]
Overall very solid code, not seeing any obvious errors in functionality.
However, I do notice a lot of unnecessary spacing which makes the code look messy.
When assigning a variable or declaring a function, having those spaces are fine but inside element properties, it is common to not include those spaces. 

for example
` `

compared to
` `

it is a bit nitpicky but these kind of things do matter to alot of developers

**Pull request merged**